### DISCUSSION TITLE
Questions on <Table>

### DISCUSSION THREAD

**Discussion by ovaisa** [iTwin/itwinjs-core#2392] [Q&A]
1) I am using the Table component with filters active.  I have graphics markers in the viewport corresponding to each row in the table.  When the user uses filters, I would like to update my graphics to show only the markers corresponding to rows that are shown in the table after filters are applied.  Can you please guide how I can get list of the filtered rows.

2) I am using buttonType={ButtonType.Blue} option for all buttons in the UI.  The filter window uses the default green buttons.  Is there a way to change these buttons to ButtonType.Blue

![Table](https://user-images.githubusercontent.com/83007711/135496775-47e63661-e773-4fb0-9666-929ed0d98947.JPG)

**Comment by NancyMcCallB** [comment]
1. If you are using your own TableDataProvider, your TableDataProvider.applyFilterDescriptors method is doing the filtering of rows, so you know in that method what the filtered rows are. If you're using another TableDataProvider, there is an internal prop on the Table named `onApplyFilter`. You can then use TableDataProvider.getRow() and .getRowsCount() to get the filtered rows. We will expose `onApplyFilter` in a future release.
2. We will look into the use of the Green vs. Blue buttons in the Filtering dialog.
